docs/index.md

```diff
@@ -1,5 +1,74 @@
 # Project overview
 
+## Current work
+
+We do sprint planning and track day-to-day work on our [Project Board][board].
+
+## Product roadmap
+
+Our product roadmap captures what we're currently building, what we've built, and what we plan to build in the future. We update it at the end of each quarter or when priorities change.
```

Comment:
this is subjective, but i think it would be less disorienting if 'Current work' and 'Product roadmap' were shuffled toward the bottom of the landing page. perhaps immediately above 'Practices'?